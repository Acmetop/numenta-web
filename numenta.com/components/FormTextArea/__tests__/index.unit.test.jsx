import React from 'react'
import renderer from 'react-test-renderer'

import FormTextArea from '../../FormTextArea'


describe('<FormTextArea /> React component unit test suite', () => {

  it('Renders correctly', () => {
    const component = renderer.create(
      <FormTextArea />
    )
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })

})
