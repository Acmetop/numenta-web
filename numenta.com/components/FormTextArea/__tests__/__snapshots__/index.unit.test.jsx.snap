exports[`<FormTextArea /> React component unit test suite Renders correctly 1`] = `
<textarea
  className="formTextArea"
  id={undefined}
  name={undefined}
  placeholder={undefined} />
`;
